src/audio-worklet/migration/spn-recorder/app.js

```diff
@@ -15,20 +15,26 @@ const WAVEFROM_SCALE_FACTOR = 5
 let recordingLength = 0;
 let recordBuffer = [[], []];
 let isRecording = false;
-let isMonitoring = false;
-let visualizationEnabled = true;
+let initCount = 0;
 
 let recordButton = document.querySelector('#record');
 let recordText = document.querySelector('#record-text');
+let stopButton = document.querySelector('#stop');
 let player = document.querySelector('#player');
 let downloadButton = document.querySelector('#download-button');
 let downloadLink = document.querySelector('#download-link');
 
 // Wait for user interaction to initialize audio, as per specification.
-document.addEventListener('click', (element) => {
-  init();
-  document.querySelector('#click-to-start').remove();
-}, {once: true});
+if (!initCount){
+  recordButton.disabled = false;
+  recordButton.addEventListener('click', (element) => {
+    init();
+    isRecording = true;
+    initCount++;
+    recordText.textContent = "Continue";
+    changeButtonDisabled();
+  }, {once: true});
```

Comment:
Yes, it is correct

Reply:
Then what's the meaning of `initCount++`? It will always be 1 after the button press.

As we discussed, using enums for different phases will be easier to read and understand.